src/Login/Login.test.js

```diff
@@ -6,43 +6,39 @@ import Login from './Login';
 import { MemoryRouter } from 'react-router-dom';
 
 describe('Login', () => {
-//Tests:
-    //handleInputChange(event)
-    //handleLogin is fired (userEvent) when 
-        //submit button is clicked
-    //can we/should we test componentDidMount?
+
     it('should render a login form', () => {
-        // const value1 = 'charlie@turing.io'
-        // const value2 = 'qwerty'
         const fakeUpdateLogin = jest.fn();
-        const fakeHandleChange = jest.fn();
-        const handleLogin = jest.fn();
-        const myUser = {}; 
+        const fakeSetUser = jest.fn();
+        const myUser = {};
 
       render(
         <MemoryRouter>
-          <Login 
+          <Login
             user={myUser}
             updateLoginView={fakeUpdateLogin}
-            // handleInputChange={fakeHandleChange}
```

Comment:
We removed line 27 because we weren't sure if or how to test methods existing on the class component itself (a method not passed as props).